torchvision/prototype/features/_feature.py

```diff
@@ -10,6 +10,11 @@
 F = TypeVar("F", bound="_Feature")
 
 
+# Due to torch.jit.script limitation we keep DType as torch.Tensor
+# instead of Union[torch.Tensor, PIL.Image.Image, features._Feature]
+DType = torch.Tensor
```

Comment:
What is the difference between this and `ImageType` in the next module? Should we have a cleanup PR that merges the two?

Reply:
`DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]` Aka can be bboxes, masks etc
`ImageType = Union[torch.Tensor, PIL.Image.Image, features.Image]` Aka only Image types (and soon Videos).

It's debatable if these aliases we use here make any sense IMO, especially if we can't define the actual types. I maintained this to keep track of what we support and where (else this info will be lost and we would have to review all signatures again) but I think we should make a decision before the cleanup PR. I would be OK removing this info in favour of `torch.Tensor` but I don't have strong opinion.

Reply:
> `DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]` Aka can be bboxes, masks etc
> `ImageType = Union[torch.Tensor, PIL.Image.Image, features.Image]` Aka only Image types (and soon Videos).

Thanks I missed that.

> I would be OK removing this info in favour of torch.Tensor (fake type) but I don't have strong opinion.

I think having such a type alias is a good thing, because it adds context. Without it we would either need to put such a comment onto each function or leave it out.

Reply:
That works for me. It just creates even more things we need to name properly. For example, `ImageType` is not a good name because soon we will add support of Videos. :( I do agree with you that it provides context on what we should be supporting and might make it easier to fix on the future if adds support of Subclassing. I don't know what's best here. @vfdev-5 thoughts?

Reply:
Let's keep these aliases